fastapi/routing.py

```diff
@@ -1355,6 +1355,8 @@ def read_users():
                 self.add_websocket_route(
                     prefix + route.path, route.endpoint, name=route.name
                 )
+            elif isinstance(route, routing.Mount):
```

Comment:
```suggestion
            elif isinstance(route, routing.Mount) and isinstance(
                route.app, StaticFiles
            ):
```
I'm not sure we should add this for any `mount`. Mounted apps can also have sub-apps. If we want to support this I think we should add much more tests.
I suggest we only support this for `StaticFiles` for now